### ISSUE TITLE
No HTTP/TCP Traffic Metrics in Kiali Graph

### ISSUE THREAD

**Issue by erSitzt** [rancher/rancher#28784]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**

- current Rancher HA Installation 2.4.7 / 2.4.8
- istio enabled
- istio workloads deployed
- some traffic on the workload ( 80 pps in the first screenshot )

**Result:**
For the last hour
![image](https://user-images.githubusercontent.com/1625376/92479481-e29c8480-f1e3-11ea-967a-6cab35f2174e.png)
or
![image](https://user-images.githubusercontent.com/1625376/92479367-ba148a80-f1e3-11ea-9efb-38f31f29bcb4.png)
or
![image](https://user-images.githubusercontent.com/1625376/92479969-8ab24d80-f1e4-11ea-8c65-b2e0516bbc6e.png)

Result for the metrics query
```
{
"metrics": {
"request_count": {
"matrix": null
},
"request_error_count": {
"matrix": null
}
},
"histograms": {
"request_duration": {
"0.5": {
"matrix": null
},
"0.95": {
"matrix": null
},
"0.99": {
"matrix": null
},
"avg": {
"matrix": null
}
},
"request_duration_millis": {
"0.5": {
"matrix": null
},
"0.95": {
"matrix": null
},
"0.99": {
"matrix": null
},
"avg": {
"matrix": null
}
}
}
}
```

**Other details that may be helpful:**

Im not sure, but i think the same problem applies to Grafana, the graphs there look mostly empty too.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):

2.4.7 / 2.4.8

- Installation option (single install/HA):

Rancher HA Installation 

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
Imported, but happening in both imported and vSphere Provider
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
VM
- Kubernetes version (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.8", GitCommit:"ec6eb119b81be488b030e849b9e64fda4caaf33c", GitTreeState:"clean", BuildDate:"2020-03-12T20:52:22Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by erSitzt** [comment]
Grafana seems to be using `istio_request_duration_milliseconds_bucket` which does not return anything in my prometheus

**Comment by erSitzt** [comment]
Could this be a similar issue to this one ?

https://github.com/weaveworks/flagger/issues/478

**Comment by erSitzt** [comment]
Anyone ?

**Comment by erSitzt** [comment]
Has nobody run into this issue ?
Are there still updates coming to the istio chart for 2.4.8 ?

**Comment by erSitzt** [comment]
Seems this was an issue which is fixed with kiali 1.18v

https://github.com/kiali/kiali/issues/2721

Can you update the istio system-chart ?

**Comment by erSitzt** [comment]
I tested v1.18 using the quai.io image and the graphs are working again...
But Kiali now has a component_status feature which fails, because the default app-label/namespace config differs from Ranchers Istio/Monitoring deployment.

maybe just disable component_status for now or provide a matching config ?
